### PR TITLE
Support produce message to pulsar

### PR DESCRIPTION
**Pull request by fu-turer**

### CODE REVIEW

File: lib/api/pulsar/pulsar_partitioned_topic_api.dart [hunk 1]
```diff
@@ -245,4 +245,54 @@ class PulsarPartitionedTopicApi {
     return new PulsarPartitionedTopicBaseResp(
         topicName, partitionNum, msgRateIn, msgRateOut, msgInCounter, msgOutCounter, storageSize);
   }
+
+  static Future<String> sendMsgToPartitionTopic(String host, int port, String tenant,
+      String namespace,
+      String topic,
+      String key,
+      String value) async {
+    String producerName = "flutter-dashboard-producer";
```
Comment (lovehzj) [inline lib/api/pulsar/pulsar_partitioned_topic_api.dart:254]: extract to `PulsarConst` ?

File: lib/api/pulsar/pulsar_partitioned_topic_api.dart [hunk 2]
```diff
@@ -245,4 +245,54 @@ class PulsarPartitionedTopicApi {
...
+
+  Map toJson() {
+    Map map = new Map();
+    map["producerName"] = this.producerName;
+    map["messages"] = this.messages;
+    return map;
+  }
+}
+
+class ProducerMessage {
```
Comment (lovehzj) [inline lib/api/pulsar/pulsar_partitioned_topic_api.dart:287]: we define dto in module package

File: lib/ui/pulsar/screen/pulsar_partitioned_topic.dart [hunk 3]
```diff
@@ -13,6 +13,9 @@ import 'package:paas_dashboard_flutter/vm/pulsar/pulsar_partitioned_topic_subscr
 import 'package:paas_dashboard_flutter/vm/pulsar/pulsar_partitioned_topic_view_model.dart';
 import 'package:provider/provider.dart';
 
+import 'package:paas_dashboard_flutter/vm/pulsar/pulsar_partitioned_topic_produce_view_model.dart';
```
Comment (lovehzj) [inline lib/ui/pulsar/screen/pulsar_partitioned_topic.dart:16]: import order looks weird

File: lib/ui/pulsar/widget/pulsar_partitioned_topic_produce.dart [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  State<StatefulWidget> createState() {
+    return new PulsarPartitionedTopicProduceWidgetState();
+  }
+}
+
+class PulsarPartitionedTopicProduceWidgetState extends State<PulsarPartitionedTopicProduceWidget> {
+  @override
+  void initState() {
+    super.initState();
+    final vm = Provider.of<PulsarPartitionedTopicProduceViewModel>(context, listen: false);
```
Comment (lovehzj) [inline lib/ui/pulsar/widget/pulsar_partitioned_topic_produce.dart:21]: unused var

File: lib/vm/pulsar/pulsar_partitioned_topic_produce_view_model.dart [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+import 'package:paas_dashboard_flutter/vm/base_load_list_view_model.dart';
+
+class PulsarPartitionedTopicProduceViewModel extends BaseLoadListViewModel {
+  final PulsarInstancePo pulsarInstancePo;
+  final TenantResp tenantResp;
+  final NamespaceResp namespaceResp;
+  final TopicResp topicResp;
+
+  PulsarPartitionedTopicProduceViewModel(this.pulsarInstancePo, this.tenantResp, this.namespaceResp, this.topicResp);
+
```
Comment (lovehzj) [inline lib/vm/pulsar/pulsar_partitioned_topic_produce_view_model.dart:15]: one space line is enough

File: lib/ui/util/form_util.dart [hunk 6]
```diff
@@ -95,6 +95,57 @@ class FormUtil {
         child: Text('Create ' + resourceName));
   }
 
+  static ButtonStyleButton createButton2Single(
```
Comment (lovehzj) [inline lib/ui/util/form_util.dart:98]: what's button single means?

File: pubspec.lock [hunk 7]
```diff
@@ -219,6 +219,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "0.12.11"
+  material_color_utilities:
```
Comment (lovehzj) [inline pubspec.lock:222]: why add this depdency?

File: pubspec.lock [hunk 8]
```diff
@@ -310,13 +317,6 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.2.1"
-  redis:
-    dependency: "direct main"
-    description:
-      name: redis
-      url: "https://pub.dartlang.org"
```
Comment (lovehzj) [inline pubspec.lock:317]: why delete this ?

**Review by goflutterjava — APPROVED** [review]

**Pull request merged**